src/plugins/syscalls/syscalls.cpp

```diff
@@ -149,6 +240,30 @@ static GSList* create_trap_config(drakvuf_t drakvuf, syscalls *s, symbols_t *sym
 
             PRINT_DEBUG("[SYSCALLS] Adding trap to %s\n", symbol->name);
 
+            syscall_wrapper_t *wrapper = (syscall_wrapper_t *)g_malloc(sizeof(syscall_wrapper_t));
+
+            wrapper->syscall_index = -1;
+
+            for (j=0; j<NUM_SYSCALLS; j++) {
+              if(strcmp(symbol->name,win_syscall_struct[j].name)==0) {
+                wrapper->sc=s;
+                wrapper->syscall_index=j;
+              }
+            }
+
+            // These functions are not found:
+            // ERROR: NtFlushProcessWriteBuffers not found
+            // ERROR: NtGetCurrentProcessorNumber not found
+            // ERROR: NtGetEnvironmentVariableEx not found
+            // ERROR: NtIsSystemResumeAutomatic not found
+            // ERROR: NtQueryEnvironmentVariableInfoEx not found
```

Comment:
We should probably find the definitions for these functions and add them to the list. Check out https://github.com/mirror/processhacker/blob/master/2.x/trunk/phlib/include/ntkeapi.h for example!